packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionsTableClaimableRow.tsx

```diff
@@ -158,14 +172,22 @@ function ClaimableRowTime({ tx }: CommonProps) {
   const destinationLayer = tx.isWithdrawal ? parentLayer : layer
   const { remainingTime } = useRemainingTime(tx)
 
+  if (tx.status === 'pending') {
+    return tx.isCctp ? <>{remainingTime}</> : <WithdrawalCountdown tx={tx} />
+  }
```

Comment:
i might have missed this but what is currently happening on prod to the pending non-cctp tx?